tests/test_series.py

```diff
@@ -1264,9 +1307,9 @@ def test_types_dot() -> None:
     s2 = pd.Series([-1, 2, -3, 4])
     df1 = pd.DataFrame([[0, 1], [-2, 3], [4, -5], [6, 7]])
     n1 = np.array([[0, 1], [1, 2], [-1, -1], [2, 0]])
-    check(assert_type(s1.dot(s2), Scalar), np.int64)
-    check(assert_type(s1 @ s2, Scalar), np.int64)
-    check(assert_type(s1.dot(df1), "pd.Series[int]"), pd.Series, np.int64)
+    check(assert_type(s1.dot(s2), Scalar), np.integer)
+    check(assert_type(s1 @ s2, Scalar), np.integer)
+    check(assert_type(s1.dot(df1), "pd.Series[int]"), pd.Series, np.integer)
```

Comment:
These are inconsistent, and I think the PYI has to be fixed.  Because if we are doing `Series[int].dot(Series[int])`, result should be `int`.  And `s1.dot(df1)` should be `Series[float]` not `Series[int]`, because we don't know the types inside of the dataframe.

Reply:
Will add it to the other issue and we can investigate deeper to avoid having too many changes in this one as there are already quite a few).